src/main/java/org/icgc/argo/program_service/mappers/ProgramMapper.java

```diff
@@ -21,7 +21,17 @@
     Program ProgramEntityToProgram(ProgramEntity entity);
     ProgramEntity ProgramToProgramEntity(Program program);
 
-    default UUID map(String s) {
+    default GetProgramResponse toGetProgramResponse(ProgramEntity entity) {
```

Comment:
i notices this interface is annotated with 
```java
@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE)
```

IGNORE should never be used, as it defeats the purpose of mapstruct. Using the current code, if a field was added to the proto file, but not to the entity, it would NOT be caught. With `ReportingPolicy.ERROR` set, we are certain that this will not go through. 

To be as defensive as possible, please change the annotation to be 
```java
@Mapper(config =  ProgramServiceMapperConfig.class)
```
and for the errors the result, refer to the link below on how to ignore target fields

https://github.com/rtisma/protobuf-mapstruct-sandbox/blob/master/src/main/java/com/roberttisma/sandbox/protobuf_mapstruct/converters/RegularConverter.java#L23-L34

I am aware that adding these annotations is messy and tedious, however, there is a ticket https://github.com/icgc-argo/program-service/issues/41 to address this issue